.clang-format

```diff
@@ -0,0 +1,76 @@
+# From https://gist.github.com/Astro36/ef933be73050b4d5a6e0522536723a18
```

Comment:
I'm assuming this is for formatting glsl? For now, I'd like to remove this as its not part of our CI and I like to keep the root of the repo as clean as possible.

Reply:
Yes. I kind of felt the same about having it there, but I wanted to at least show it. The config I currently use is set up to approximate rust-fmt. Would you be interested in having it in CI and if so, because I'm happy to do a PR for that too.

Where would you want the file to live? The VSCode plugin I use (`Shader languages support for VS Code`), for example, needs it to be in an ancestor directory of the shaders you want to have formatted.

Reply:
I think for now its more trouble than its worth. Consistent shader formatting isn't something that has come up yet as a major issue and it makes the validation pipeline more complicated. The "official shader language" might also potentially change (ex: WGSL or rust-gpu). Lets revisit when we have more shaders, the renderer has stabilized a bit, and consistency is a bigger concern.

Reply:
I've removed `.clang-format`.